dnf-behave-tests/fixtures/specs/dnf-ci-scriptlets/Package-filetriggerin-fail.spec

```diff
@@ -0,0 +1,19 @@
+Name:           Package-filetriggerin-fail
```

Comment:
Should the repository be named just `scriptlets` to go with the convention we have?